app/models/katello/glue/elastic_search/product.rb

```diff
@@ -1,37 +1,5 @@
 module Katello
   module Glue::ElasticSearch::Product
```

Comment:
I suppose I can live with that, my reasons for moving them (which I did similarly on another conversion) were:

1) Get rid of this module all together
2) The method call would remain once those got coverted (package/PuppetModule) but the guys would be the only thing needing to change.